docs/landing_pages/owasp.md

```diff
@@ -0,0 +1,37 @@
+---
+title: EEF Security WG and OWASP
+layout: page
+---
+
+[![OWASP][owasp_logo]][owasp_website]
+
+The EEF Security Working Group and [OWASP][owasp_website] share a mission to
+encourage secure software development practices and to share practical,
+actionable knowledge with developers worldwide. While OWASP resources tend to
+be platform agnostic, the EEF Security WG focusses on the Erlang ecosystem,
+making the provided resources very much complementary.
```

Comment:
```suggestion
making the provided resources very highly complementary.
```

Reply:
I think without very, in that case, no?

Reply:
Yes. I was intending without very, but messed up the suggestion.